src/strategies/erc721-with-tokenid-weighted-types/index.ts

```diff
@@ -0,0 +1,57 @@
+import { multicall } from '../../utils';
+
+export const author = 'gregegan';
+export const version = '0.1.0';
+
+const abi = [
+  'function ownerOf(uint256 tokenId) public view returns (address owner)'
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+  const responsePrimary = await multicall(
+    network,
+    provider,
+    abi,
+    options.tokenIdsPrimary.map((id: any) => [
```

Comment:
Hello @gregegan Instead of passing all tokenIDs into strategy params, would be great if we can do it like ranges like in `erc721-with-tokenid-range-weights` strategy 

Also, the name can be similar to that too, maybe like `erc721-with-tokenid-range-weights-v2` or `erc721-tokenid-range-weights-simple` or other